ratpack-session/src/main/java/ratpack/session/AllowedAnnotationSessionTypeFilterPlugin.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ratpack.session;
+
+public class AllowedAnnotationSessionTypeFilterPlugin implements SessionTypeFilterPlugin {
+
+  @Override
+  public boolean allow(String className) {
+    return false;
+  }
+}
```

Comment:
I'm confused what this is doing. It seems like it just forbids all classes? Some documentation explaining what's going on here would be helpful.

Reply:
Relic. Now removed.